### DISCUSSION TITLE
Where did ScrollView go?

### DISCUSSION THREAD

**Discussion by jeffwright13** [Textualize/textual#356] [Q&A] [answered]
I've been messing with the `css` branch and noticed that the `ScrollView` widget is gone. Is that intentional? And if so, is there a recommended replacement widget?

**Comment by tusharsadhwani** [comment, marked as answer]
Check the source of examples/borders.py in the css branch :)

I think eventually the `overflow-y: scroll` is what will trigger the vertical scrolling, but right now it seems it's a flag

**Reply by willmcgugan** [reply]
Any widget can effectively become a `ScrollView` now.

**Comment by jeffwright13** [comment]
THanks for the replies :-)
